lib/Rex/Config.pm

```diff
@@ -99,6 +127,16 @@ sub get_use_rex_kvm_agent {
   return $use_rex_kvm_agent;
 }
 
+=head2 set_use_template_ng
+
+=head2 get_use_template_ng
+
+Sets and gets the value of the C<$use_template_ng> configuration variable.
+
+This controls whether Rex should use L<Rex::Template::NG> to render templates.
```

Comment:
As `Rex::Template::NG` seems to be undocumented, what does this mean?

Reply:
`Rex::Template::NG` should get its own docs for the true meaning of the effects.

Strictly speaking, the config option here is about whether it's on or off. So in the context of `Rex::Config`, I just tried to stick to document that behavior (but already linked to `Rex::Template::NG` despite having no docs there yet).